### PR TITLE
Convert entityInfo into a JsonObject inside logAuditMessage

### PR DESCRIPTION
**Pull request by IsuruBoyagane15**
`9.22.x` changes of the change to `9.2.x` by  https://github.com/wso2/carbon-apimgt/pull/11331 and https://github.com/wso2/carbon-apimgt/pull/11335

Related: https://github.com/wso2-enterprise/choreo/issues/14247

### CODE REVIEW

**Comment by Jayanie** [comment]
Is the change introduced by the PR https://github.com/wso2/carbon-apimgt/pull/11331 included in this?

**Comment by IsuruBoyagane15** [comment]
> Is the change introduced by the PR #11331 included in this?

Yes. Some changes added in #11331 were removed with the #11335.
The sum of those two changes are only added in this PR to 9.22.x.

**Comment by Jayanie** [comment]
> > Is the change introduced by the PR #11331 included in this?
> 
> Yes. Some changes added in #11331 were removed with the #11335. The sum of those two changes are only added in this PR to 9.22.x.

Ack

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java [hunk 1]
```diff
@@ -6812,7 +6812,12 @@ public static void logAuditMessage(String entityType, String entityInfo, String
         jsonObject.put("typ", entityType);
         jsonObject.put("action", action);
         jsonObject.put("performedBy", performedBy);
-        jsonObject.put("info", entityInfo);
+        try {
+            JSONObject entityInfoJson = (JSONObject) new JSONParser().parse(entityInfo);
```
Comment (ShalkiWenushika) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java:6816]: Have we convert the entityInfoJson into a JSONObject? can see the type as string in here: https://github.com/wso2/carbon-apimgt/blob/9.22.x/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java#L6809

**Review by ShalkiWenushika — APPROVED** [review]

**Review by Jayanie — APPROVED** [review]

**Pull request merged**